src/app/organization/bulk-import/bulk-import.component.html

```diff
@@ -65,6 +65,13 @@ <h4 matLine>Chart of Accounts</h4>
             <h4 matLine>Share Accounts</h4>
           </mat-list-item>
 
+          <mat-list-item [routerLink]="['repayments-download']" *mifosxHasPermission="'READ_CLIENT'">
+            <mat-icon matListIcon>
+              <fa-icon icon="money-bill-alt" size="sm"></fa-icon>
+            </mat-icon>
+            <h4 matLine>Repayments Download</h4>
```

Comment:
quick question @william000000 : do we have translation setup in v2? If yes, it would be better to use translation key instead of literals